allure-report-face/src/styles.css

```diff
@@ -56,6 +60,7 @@ html, body {
 }
 
 .long-line {
+  line-height: 1.5em;
```

Comment:
Yes. You can add line height to `.table__col` class if you would like to. This is a better place for it.